backend/tests/election_integration_test.rs

```diff
@@ -68,4 +69,19 @@ async fn test_election_details_status(pool: SqlitePool) {
     assert_eq!(status, StatusCode::OK);
     assert!(!body.statuses.is_empty());
     assert_eq!(body.statuses[0].status, PollingStationStatus::Incomplete);
+    assert_eq!(body.statuses[1].status, PollingStationStatus::Incomplete);
+
+    shared::create_and_finalise_data_entry(&addr).await;
+
+    let url = format!("http://{addr}/api/elections/1/status");
+    let response = reqwest::Client::new().get(&url).send().await.unwrap();
+    let status = response.status();
+    let body: ElectionStatusResponse = response.json().await.unwrap();
+
+    // Ensure the response is what we expect
+    println!("response body: {:?}", &body);
+    assert_eq!(status, StatusCode::OK);
+    assert!(!body.statuses.is_empty());
+    assert_eq!(body.statuses[0].status, PollingStationStatus::Complete);
+    assert_eq!(body.statuses[1].status, PollingStationStatus::Incomplete);
```

Comment:
The SQL query doesn't sort by polling station id so I think the order of the id's in the response vector is not guaranteed?

Reply:
I don't know how sqlite does this, but let's not risk a flaky test :)
[1c644a5](https://github.com/kiesraad/abacus/pull/238/commits/1c644a5a53ea704205f154afee63c263621ed356) solves this